### PR TITLE
support space in workspace

### PR DESCRIPTION
**Pull request by jolorunyomi**
# Summary
This PR adds support for spaces in workspace

### CODE REVIEW

File: ci/docs/build.sh [hunk 1]
```diff
@@ -43,13 +43,13 @@ cd $PROJECT_WORKSPACE/docs
...
 #Commit to Website
-cd $DOCS_WORKSPACE
+cd "$DOCS_WORKSPACE"
 
 for PROJECT in ${PROJECTS[@]}; do
     if [ ! -d "api/$PROJECT/$BRANCH_VERSION" ]; then
         mkdir -p api/$PROJECT/$BRANCH_VERSION
     fi
-    rm -rf $DOCS_WORKSPACE/api/$PROJECT/$BRANCH_VERSION/*	
+    rm -rf "$DOCS_WORKSPACE/api/"$PROJECT/$BRANCH_VERSION/*
```
Comment (jolorunyomi) [inline ci/docs/build.sh:52]:
```suggestion
    rm -rf "$DOCS_WORKSPACE/api/$PROJECT/$BRANCH_VERSION/*	
```

File: ci/docs/build.sh [hunk 2]
```diff
@@ -43,13 +43,13 @@ cd $PROJECT_WORKSPACE/docs
...
 for PROJECT in ${PROJECTS[@]}; do
     if [ ! -d "api/$PROJECT/$BRANCH_VERSION" ]; then
         mkdir -p api/$PROJECT/$BRANCH_VERSION
     fi
-    rm -rf $DOCS_WORKSPACE/api/$PROJECT/$BRANCH_VERSION/*	
+    rm -rf "$DOCS_WORKSPACE/api/"$PROJECT/$BRANCH_VERSION/*	
 done
 
-mv $PROJECT_WORKSPACE/docs/_build/html/* $DOCS_WORKSPACE/api/cuxfilter/$BRANCH_VERSION
+mv $PROJECT_WORKSPACE/docs/_build/html/* "$DOCS_WORKSPACE/api/cuxfilter/"$BRANCH_VERSION
```
Comment (jolorunyomi) [inline ci/docs/build.sh:55]:
```suggestion
mv $PROJECT_WORKSPACE/docs/_build/html/* "$DOCS_WORKSPACE/api/cuxfilter/$BRANCH_VERSION
```

File: ci/docs/build.sh [hunk 3]
```diff
@@ -43,13 +43,13 @@ cd $PROJECT_WORKSPACE/docs
...
 #Commit to Website
-cd $DOCS_WORKSPACE
+cd "$DOCS_WORKSPACE"
 
 for PROJECT in ${PROJECTS[@]}; do
     if [ ! -d "api/$PROJECT/$BRANCH_VERSION" ]; then
         mkdir -p api/$PROJECT/$BRANCH_VERSION
     fi
-    rm -rf $DOCS_WORKSPACE/api/$PROJECT/$BRANCH_VERSION/*	
+    rm -rf "$DOCS_WORKSPACE/api/$PROJECT/$BRANCH_VERSION/*
```
Comment (ajschmidt8) [inline ci/docs/build.sh:52]:
```suggestion
    rm -rf "$DOCS_WORKSPACE/api/$PROJECT/$BRANCH_VERSION/"*	
```
looks like this is missing a closing `"`

File: ci/docs/build.sh [hunk 4]
```diff
@@ -43,13 +43,13 @@ cd $PROJECT_WORKSPACE/docs
...
 for PROJECT in ${PROJECTS[@]}; do
     if [ ! -d "api/$PROJECT/$BRANCH_VERSION" ]; then
         mkdir -p api/$PROJECT/$BRANCH_VERSION
     fi
-    rm -rf $DOCS_WORKSPACE/api/$PROJECT/$BRANCH_VERSION/*	
+    rm -rf "$DOCS_WORKSPACE/api/$PROJECT/$BRANCH_VERSION/*	
 done
 
-mv $PROJECT_WORKSPACE/docs/_build/html/* $DOCS_WORKSPACE/api/cuxfilter/$BRANCH_VERSION
+mv $PROJECT_WORKSPACE/docs/_build/html/* "$DOCS_WORKSPACE/api/cuxfilter/$BRANCH_VERSION
```
Comment (ajschmidt8) [inline ci/docs/build.sh:55]:
```suggestion
mv $PROJECT_WORKSPACE/docs/_build/html/* "$DOCS_WORKSPACE/api/cuxfilter/$BRANCH_VERSION"
```

looks like this is missing a closing `"`

**Review by ajschmidt8 — CHANGES_REQUESTED** [review]

File: ci/docs/build.sh (hunk 3 above)
Comment (ajschmidt8) [reply ci/docs/build.sh:52]: @jolorunyomi, I still don't see a closing `"` here.

File: ci/docs/build.sh (hunk 4 above)
Comment (ajschmidt8) [reply ci/docs/build.sh:55]: @jolorunyomi, I still don't see a closing `"` here.

**Review by ajschmidt8 — CHANGES_REQUESTED** [review]

File: ci/docs/build.sh [hunk 5]
```diff
@@ -43,13 +43,13 @@ cd $PROJECT_WORKSPACE/docs
...
 for PROJECT in ${PROJECTS[@]}; do
     if [ ! -d "api/$PROJECT/$BRANCH_VERSION" ]; then
         mkdir -p api/$PROJECT/$BRANCH_VERSION
     fi
-    rm -rf $DOCS_WORKSPACE/api/$PROJECT/$BRANCH_VERSION/*	
+    rm -rf "$DOCS_WORKSPACE/api/$PROJECT/$BRANCH_VERSION/"*	
 done
 
-mv $PROJECT_WORKSPACE/docs/_build/html/* $DOCS_WORKSPACE/api/cuxfilter/$BRANCH_VERSION
+mv $PROJECT_WORKSPACE/docs/_build/html/* "$DOCS_WORKSPACE/api/cuxfilter/$BRANCH_VERSION
```
Comment (jolorunyomi) [inline ci/docs/build.sh:55]:
```suggestion
mv $PROJECT_WORKSPACE/docs/_build/html/* "$DOCS_WORKSPACE/api/cuxfilter/$BRANCH_VERSION"
```

File: ci/docs/build.sh [hunk 6]
```diff
@@ -43,13 +43,13 @@ cd $PROJECT_WORKSPACE/docs
...
 for PROJECT in ${PROJECTS[@]}; do
     if [ ! -d "api/$PROJECT/$BRANCH_VERSION" ]; then
         mkdir -p api/$PROJECT/$BRANCH_VERSION
     fi
-    rm -rf $DOCS_WORKSPACE/api/$PROJECT/$BRANCH_VERSION/*	
+    rm -rf "$DOCS_WORKSPACE/api/$PROJECT/$BRANCH_VERSION/"*	
 done
 
-mv $PROJECT_WORKSPACE/docs/_build/html/* $DOCS_WORKSPACE/api/cuxfilter/$BRANCH_VERSION
+mv $PROJECT_WORKSPACE/docs/_build/html/* "$DOCS_WORKSPACE/api/cuxfilter/$BRANCH_VERSION"
```
Comment (jolorunyomi) [inline ci/docs/build.sh:55]:
```suggestion
mv "$PROJECT_WORKSPACE/docs/_build/html/"* "$DOCS_WORKSPACE/api/cuxfilter/$BRANCH_VERSION"
```

**Review by ajschmidt8 — APPROVED** [review]

**Comment by ajschmidt8** [comment]
@gpucibot merge

**Pull request merged**